### ISSUE TITLE
 Answer sheet generator

### ISSUE THREAD

**Issue by soulcharmer** [soulcharmer/pubquiz_generator#2]
from the single-source Q&A file, we need a generator for easy preparation of the answer printout sheet to help the quiz moderators to check answers quickly.

**Comment by zelial** [comment]
How would the answer sheet look like? A mock up would be nice

**Comment by soulcharmer** [comment]
If you have something like:


1-1_q1: |
        How do you say "emergency" in German? <<<Definite article required for full answer.
1-1_a1: |
        die Not

1-1_q2: |
        How do you say "food" in Spanish?
1-1_a2: |
        alimento


then the output would flattened as plain text like this:


1-1_q1: How do you say "emergency" in German? **die Not**
1-1_q2: How do you say "food" in Spanish?  **alimento**


The "<<<" sequence above is a cut-off token to mark the end of the question text to be reused in the answer sheet AS WELL AS answer slides. (Do we need a separate ticket to implement the cut-off token for answer slides?) 

The ** markup denotes bold text in a wiki syntax.

Both of these special sequences can be changed for something more convenient.

Thanks!

**Comment by zelial** [comment]
So no formatting except for bold text? Should be possible with some simple pdf generating library. /me needs to look into this

**Comment by soulcharmer** [comment]
Correct, no need to make the answer sheet any fancier (at least not for now :)
